docs/reference/edot-sdks/nodejs/migration.md

```diff
@@ -162,6 +212,10 @@ The Elastic APM Node.js agent [`metricsInterval`](apm-agent-nodejs://reference/c
 
 For example: `OTEL_METRIC_EXPORT_INTERVAL=30000`.
 
+### `opentelemetryBridgeEnabled`
+
+If you were using the Elastic APM Node.js agent [`opentelemetryBridgeEnabled`](apm-agent-nodejs://reference/configuration.md#opentelemetry-bridge-enabled) option you can drop it safely and keep using `@openetelemetry/api` to manually instrument your service.
+
```

Comment:
I'm torn. Perhaps just don't mention this?  The OTel Bridge was never GA. I'm fine either way.